cmd/queue/main.go

```diff
@@ -292,11 +292,15 @@ func buildServer(ctx context.Context, env config, healthState *health.State, rp
 	breaker := buildBreaker(logger, env)
 	metricsSupported := supportsMetrics(ctx, logger, env)
 	tracingEnabled := env.TracingConfigBackend != tracingconfig.None
+	concurrencyStateEnabled := env.ConcurrencyStateEndpoint != ""
 	timeout := time.Duration(env.RevisionTimeoutSeconds) * time.Second
 
 	// Create queue handler chain.
 	// Note: innermost handlers are specified first, ie. the last handler in the chain will be executed first.
 	var composedHandler http.Handler = httpProxy
+	if concurrencyStateEnabled {
```

Comment:
So the config string don't need to "freeze-proxy"? according to: https://github.com/knative/serving/pull/11699/files

Reply:
No, the config string should be the service endpoint of the freeze service and its port. For example, assuming there was a `freeze-service` running in the `knative-serving` namespace using port `9696`, the config value could be: `freeze-service.knative-serving.svc.cluster.local:9696`.

Reply:
Got it